test-suite/src/data/schema.ts

```diff
@@ -222,94 +222,95 @@ const schema: Schema = {
         },
         {
           testName: "Safeguarding Referral",
-          testDescription: "This test will attempt to send notification of a safeguarding referral to Social Services.",
+          testDescription:
+            "This test will attempt to send notification of a safeguarding referral to Social Services.",
           template: require("./safeguarding-request.xml"),
           testSpecifications: {
             [RequestHeaderProps.Header]: [
               {
                 label: "url",
                 id: "url",
                 defaultValue: "http://localhost:8081/report",
-                validators: [isUrl(), isReportUrl()]
+                validators: [isUrl(), isReportUrl()],
               },
               {
                 label: "Content Type",
                 id: "content-type",
                 defaultValue: "application/xml",
-                validators: [notNull(), minLength(7)]
-              }
+                validators: [notNull(), minLength(7)],
```

Comment:
We have additional commas in this json file too. Please remove.

Reply:
Changes made.